### ISSUE TITLE
[Enhancement] Implement signature detection

### ISSUE THREAD

**Issue by KoNickss** [lucanaef/PGPro#71]
Implement signature detection where, if the encrypted message is also signed, a "Valid Signature from X on *date*" type of message pops up, and maybe automatic key fetching if the signing key isnt alr in the db, similar to what commandline GPG does

**Comment by ibnaleem** [comment]
Agreed, there should also be a section that strictly checks for signed content to verify (such as detached signatures of clear-signed documents)
